bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieImpl.java

```diff
@@ -872,6 +869,9 @@ int shutdown(int exitCode) {
                 // Shutdown the EntryLogger which has the GarbageCollector Thread running
                 ledgerStorage.shutdown();
 
+                // Shutdown Sync thread
+                syncThread.shutdown();
```

Comment:
Here may through exceptions when shutdown the `syncThread` which will call `checkpoint`  of ledgerStoreage, since we have already shut down the ledgeerStorage before

Reply:
Yes, you are right. I updated the code, please help take a look, thanks.